db/projects/ripple/project.yaml

```diff
@@ -0,0 +1,107 @@
+name: XRP
+start_date: '2013-02-02'
+description: 'Ripple is a system of the real time gross settlements, exchange a money transfer system'
+web:
+  - 'https://ripple.com/xrp'
+ledgers:
+  - id: xrp
+    name: XRP
+    type: blockchain
+    networks:
+      - id: main
+        type: main
+        name: Ripple Network
+        mineable: false
+        tools:
+          - type: explorer
+            url: 'https://xrpcharts.ripple.com/#/graph'
+          - type: explorer
+            url: 'https://bithomp.com/explorer'
+assets:
+  - id: xrp
+    description: 'XRP is a public, counterparty-less asset native to the XRP Ledger, and is designed to bridge the many different currencies in use worldwide.'
+    name: XRP
+    type: native
+    ledger: xrp
+    symbol: XRP
+    max_supply: 100000000000
+    start_date: '2013-02-02'
+    webids:
+      coinmarketcap: ripple
+      cryptocompare: xrp
+      cryptocompare/id: '5031'
+webids:
+  custom/forum: 'http://www.xrpchat.com'
+  bitcoinwiki: Ripple
+  telegram: Ripple
+  discord: N4x7kPb
+  github: ripple
+  twitter: Ripple
+  reddit: ripple
+  wikipedia/en: Ripple_(payment_protocol)
```

Comment:
Maybe put it as **'**Ripple_(payment_protocol)**'**

Reply:
done